nemo/collections/tts/modules/transformer.py

```diff
@@ -127,9 +127,9 @@ def _forward(self, inp, attn_mask=None):
         head_k = head_k.view(inp.size(0), inp.size(1), n_head, d_head)
         head_v = head_v.view(inp.size(0), inp.size(1), n_head, d_head)
 
-        q = head_q.permute(0, 2, 1, 3).reshape(-1, inp.size(1), d_head)
-        k = head_k.permute(0, 2, 1, 3).reshape(-1, inp.size(1), d_head)
-        v = head_v.permute(0, 2, 1, 3).reshape(-1, inp.size(1), d_head)
+        q = head_q.permute(2, 0, 1, 3).reshape(-1, inp.size(1), d_head)
+        k = head_k.permute(2, 0, 1, 3).reshape(-1, inp.size(1), d_head)
+        v = head_v.permute(2, 0, 1, 3).reshape(-1, inp.size(1), d_head)
```

Comment:
Sorry, I don't follow--what's the purpose of this change?

Reply:
It's a bugfix - the dims are incorrect and it matters when there are > 1 attention heads.

Reply:
Gotcha, thanks.